stories/molecules/treemap-prototype/contributor-node.tsx

```diff
@@ -45,7 +43,7 @@ const NonMemoizedContributorNode = <Datum extends { id: string; value?: number;
       />
       {showLabel && (
         <animated.div
-          className="grid p-3 text-white place-items-start pointer-events-none"
+          className="grid p-3 text-white pointer-events-none place-items-start"
```

Comment:
Is prettier or eslint rearranging the classes? If not, revert the changes as they are not related to the issue.